static/app/icons/iconParenthesis.tsx

```diff
@@ -0,0 +1,35 @@
+import {forwardRef} from 'react';
+import styled from '@emotion/styled';
+
+import type {SVGIconProps} from './svgIcon';
+
+interface Props extends SVGIconProps {
+  side?: 'left' | 'right';
+}
+
+const IconParenthesis = forwardRef<SVGSVGElement, Props>(
```

Comment:
This svg comes from the [figma](https://www.figma.com/design/qLPE7uSdRZMOJloVumv8rA/Q1-Smart-Search?node-id=629-1328&t=kAquT8bb0dHT0oM0-0) and cannot be a square, so I did not use the `SVGIcon` component.